src/Kirel.Identity.Server.API/appsettings.Development.json

```diff
@@ -5,7 +5,10 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
-  "dbConfig": {
+  "ControllerState": {
```

Comment:
Why you have here changes with controller state from another PR?

Reply:
why not ?